README.md

```diff
@@ -238,11 +238,8 @@ Is used to generate data cubes.
     $ xcube gen --info
     input processors to be used with option --proc:
       default                           Single-scene NetCDF/CF inputs in xcube standard format
-      rbins-seviri-highroc-scene-l2     RBINS SEVIRI HIGHROC single-scene Level-2 NetCDF inputs
-      rbins-seviri-highroc-daily-l2     RBINS SEVIRI HIGHROC daily Level-2 NetCDF inputs
-      snap-olci-highroc-l2              SNAP Sentinel-3 OLCI HIGHROC Level-2 NetCDF inputs
-      snap-olci-cyanoalert-l2           SNAP Sentinel-3 OLCI CyanoAlert Level-2 NetCDF inputs
-      vito-s2plus-l2                    VITO Sentinel-2 Plus Level 2 NetCDF inputs
+
```

Comment:
Optional: Leave here, and say that a 5 input processor have been installed via plugins.